litellm/tests/test_caching.py

```diff
@@ -1569,3 +1569,102 @@ async def test_redis_semantic_cache_acompletion():
     )
     print(f"response2: {response2}")
     assert response1.id == response2.id
+
+
+@pytest.mark.skip(reason="beta test - new dual semantic cache")
```

Comment:
@powerhouseofthecell do you mean to skip this test?

Reply:
sorry i marked them the same way the redis semantic cache tests were marked - happy to not have them skipped!